### ISSUE TITLE
Always pop-up errors

### ISSUE THREAD

**Issue by nguyennhimbth** [Macronair/TheMillionaireGame#62]
The app always pop-up error (the Main app and QEDIT) even i install SQL
Main app error

> Unhandled exception has occurred in your application. If you click Continue, the application will ignore this error and attempt to continue. If you click Quit, the application will close immediately.

Retrieving the COM class factory for component with CLSID {6BF52A52-394A-11D3-B153-00C04F79FAA6} failed due to the following error: 80040154 Class not registered (Exception from HRESULT: 0x80040154 (REGDB_E_CLASSNOTREG)).

QEDIT
Unhandled exception has occurred in your application. If you click Continue, the application will ignore this error and attempt to continue. If you click Quit, the application will close immediately.

An attempt to attach an auto-named database for file C:\Users\duynguyendaccau\Downloads\MillionaireGame_v1.1.684\MillionaireGame_v1.1.684\dbMillionaire.mdf failed. A database with the same name exists, or specified file cannot be opened, or it is located on UNC share.

**Comment by nguyennhimbth** [comment]
hello @Macronair

**Comment by nguyennhimbth** [comment]
Can You See The email

**Comment by Macronair** [comment]
When you get this error, I recommend you install the full SQL Server Express software which is free and also included in the new version (1.2) of the application. This should fix your problem.

**Comment by k1m1nonawa4869** [comment]
Hello @Macronair , recently I had the same problem with this guy, so I chose the Custom option when installing SQL Server Express 2022. Still the problem exists, but my QEDIT shown a bit different message. What should I do? Thank you

![Image](https://github.com/user-attachments/assets/44029808-910a-42ed-8931-694da9e070a5) - this is when I opened QEDIT

![Image](https://github.com/user-attachments/assets/dff9000f-72ef-4b2b-b666-31faf7ee20bd) - this is when I opened Console
